tracer/src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -87,6 +87,9 @@ static DatadogLogging()
                     loggerConfiguration.Enrich.WithProperty("MachineName", DomainMetadata.MachineName);
                     loggerConfiguration.Enrich.WithProperty("Process", $"[{DomainMetadata.ProcessId} {DomainMetadata.ProcessName}]");
                     loggerConfiguration.Enrich.WithProperty("AppDomain", $"[{DomainMetadata.AppDomainId} {DomainMetadata.AppDomainName}]");
+#if NETCOREAPP
+                    loggerConfiguration.Enrich.WithProperty("AssemblyLoadContext", $"[{System.Runtime.Loader.AssemblyLoadContext.GetLoadContext(typeof(DatadogLogging).Assembly).Name}]");
```

Comment:
Do we need to wrap this in `[]`? I think it's only used for the above because it's a pair of values?
```suggestion
                    loggerConfiguration.Enrich.WithProperty("AssemblyLoadContext", System.Runtime.Loader.AssemblyLoadContext.GetLoadContext(typeof(DatadogLogging).Assembly).Name);
```
```suggestion
                    loggerConfiguration.Enrich.WithProperty("AssemblyLoadContext", $"[{System.Runtime.Loader.AssemblyLoadContext.GetLoadContext(typeof(DatadogLogging).Assembly).Name}]");
```

Reply:
Ah you're right, I just copied and pasted the AppDomain logging code without understanding why the `[]` was used. Thanks for the suggestion!